### PR TITLE
Added login and register logic

### PR DESCRIPTION
**Pull request by MajewskiOliwier**
Check if everything is right

### CODE REVIEW

File: src/main/java/com/example/JavaProject/config/SecurityConfig.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+                    .orElseThrow(() -> new UsernameNotFoundException("User not found with email: " + email));
+
+            return new User(user.getUsername(), user.getPassword(), user.getAuthorities());
+        };
+    }
+
+    @Bean
+    public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Exception {
+        http
+                .csrf().disable()
```
Comment (TomaszMarek) [inline src/main/java/com/example/JavaProject/config/SecurityConfig.java]: Csrf is is deprecated, check for something that is up to date!!

**Review by TomaszMarek — CHANGES_REQUESTED** [review]

File: src/main/java/com/example/JavaProject/request/RegisterRequest.java [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package com.example.JavaProject.request;
+
+public class RegisterRequest {
```
Comment (Karol2846) [inline src/main/java/com/example/JavaProject/request/RegisterRequest.java]: Are you planning on using RegisterRequest?
That class is empty

**Review by Karol2846 — CHANGES_REQUESTED** [review]

File: src/main/java/com/example/JavaProject/config/SecurityConfig.java (hunk 1 above)
Comment (MajewskiOliwier) [reply src/main/java/com/example/JavaProject/config/SecurityConfig.java]: i have replaced it with ".csrf(csrf -> csrf.disable())"

File: src/main/java/com/example/JavaProject/request/RegisterRequest.java (hunk 2 above)
Comment (MajewskiOliwier) [reply src/main/java/com/example/JavaProject/request/RegisterRequest.java]: I must have forgotten about it. It is gone now

**Review by TomaszMarek — APPROVED** [review]

**Review by Karol2846 — APPROVED** [review]

**Pull request merged**